web/template/admin/admin_tabs/server-notifications.gohtml

```diff
@@ -1,51 +1,47 @@
 {{define "server-notifications-admin-list"}}
-    <link rel="stylesheet" href="/static/node_modules/flatpickr/dist/flatpickr.min.css">
-    <script src="/static/node_modules/flatpickr/dist/flatpickr.min.js"></script>
-    <script defer async src="/static/assets/ts-dist/server-notifications.js"></script>
-    <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Notifications</h2>
-    {{range $notification := .}}
-        {{template "server-notification-edit" $notification}}
-    {{end}}
-    <h2 class="mt-6 text-3 font-medium">Create a new notification</h2>
-    {{template "server-notification-edit"}}
+<link rel="stylesheet" href="/static/node_modules/flatpickr/dist/flatpickr.min.css">
+<script src="/static/node_modules/flatpickr/dist/flatpickr.min.js"></script>
+<script defer async src="/static/assets/ts-dist/server-notifications.js"></script>
+<h2 class="text-2xl text-1 my-4">Notifications</h2>
+{{range $notification := .}}
+{{template "server-notification-edit" $notification}}
```

Comment:
Could you add `gap` to the parent of each "edit-container", so that there is a bit of space in between those? Thanks!